### PR TITLE
jet-antd v04.10: Removed margin from html elements

### PR DESCRIPTION
**Pull request by Tristyn**
Removed `margin: @spacing-xs;` from all the elements
then removed all instances of it getting unset
then fixed any big problems

This will fix some alignment problems in governance too

### CODE REVIEW

File: src/contexts/localization/languages/Jet_UI_KR.json [hunk 1]
```diff
@@ -55,7 +55,7 @@
     "oweLess": "덜 빚지고 있습니다.",
     "geobanned": "귀하의 지역에서는 Jet Protocol을 사용할 수 없습니다.",
     "noMarket": "Jet marketplace를 초기화하지 못했습니다.",
-    "txSuccess": "성공적입니다! {{AMOUNT AND ASSET}} 중 {{TRADE ACTION}}이 접수되었습니다. 거래를 확인해보세요 <a href='{{EXPLORER LINK}}' class='text-gradient' target='_blank' style='margin: unset; font-weight: bold;'>explorer</a>.",
+    "txSuccess": "성공적입니다! {{AMOUNT AND ASSET}} 중 {{TRADE ACTION}}이 접수되었습니다. 거래를 확인해보세요 <a href='{{EXPLORER LINK}}' class='text-gradient' target='_blank' style='font-weight: bold;'>explorer</a>.",
```
Comment (Tristyn) [inline src/contexts/localization/languages/Jet_UI_KR.json:58]: I know this needs to get changed in poeditor, but I made a task to remove html elements from translations

File: src/styles/App.less [hunk 2]
```diff
@@ -911,6 +895,7 @@ svg path {
         }
 
         i {
+          padding: @spacing-xs;
```
Comment (Tristyn) [inline src/styles/App.less:898]: This fixes a subtle bug where the top part of the search icon get's cut off cause 1px  of it's antialiasing stretches into its margin

**Comment by Tristyn** [comment]
There's a few very subtle changes to the app where elements have 0px margin instead of 2px but I think it's all inconsequential. I fixed every major change in margin

**Comment by Tristyn** [comment]
Need to deploy a new jet-antd for governance

**Review by callensm — APPROVED** [review]

**Pull request merged**